commercial/app/commercial/CommercialLifecycle.scala

```diff
@@ -102,13 +127,19 @@ trait CommercialLifecycle extends GlobalSettings with Logging with ExecutionCont
       val feedName = parser.feedMetaData.name
       val jobName = mkJobName(feedName, "parse")
       Jobs.deschedule(jobName)
-      Jobs.scheduleEveryNMinutes(jobName, 15) {
+      Jobs.scheduleEveryNMinutes(jobName, 1) {
         parseFeed(parser)
       }
     }
 
+    Jobs.deschedule("update-fetch-and-parse-metrics")
+    Jobs.scheduleEveryNMinutes("update-fetch-and-parse-metrics", 1){
```

Comment:
Very minor, but if the job name is the same for each, consider sticking it in a `val` - doesn't really save space but can make for easier reading of the code.